src/webapp/pages/bulk-apply/steps/ListDependenciesStep.tsx

```diff
@@ -67,7 +67,11 @@ export const ListDependenciesStep: React.FC<MetadataSharingWizardStepProps> = ({
                 text: i18n.t("Include dependency"),
                 multiple: true,
                 icon: <AddCircleOutlineIcon />,
-                isActive: (rows: MetadataItem[]) => _.every(rows, row => builder.excludedDependencies.includes(row.id)),
+                isActive: (rows: MetadataItem[]) =>
+                    _.every(
+                        rows,
+                        row => builder.excludedDependencies.includes(row.id) && !globalExclusions.includes(row.id)
```

Comment:
Remember that when you merge this feature with development and also this one https://github.com/EyeSeeTea/sharing-settings-app-dev/pull/11 you'll need to take into account in this step ListDependenciesStep globalExclusions for new features added in #11